modules/adding-node-iso-flags.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/nodes-nodes-adding-node-iso.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-node-iso-flags_{context}"]
+= Adding a node with command flags
+
+You can add a single node to your cluster by using command flags to specify configurations for the new node.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`)
+* You have an active connection to your target cluster
+* You have a kubeconfig file available
+
+.Procedure
+
+. Generate the ISO image by running the following command. The MAC address must be specified using a command flag.
```

Comment:
Could be useful to put a link to the additional flags that could be used - otherwise it may not be clear that they are available

Reply:
I can reference the name of the section here but I cannot directly link (to somewhere else in this repo) within a module, due to limitations of the way our docs get built.